gecko_client/admin_api.go

```diff
@@ -43,7 +43,7 @@ type GetNodeIDResponse struct {
 }
 
 func (api AdminApi) GetPeers() ([]Peer, error) {
-	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(adminEndpoint, "admin.peers")
+	responseBodyBytes, err := api.rpcRequester.makeRpcRequest(adminEndpoint, "admin.peers", make(map[string]interface{}))
```

Comment:
you'll need to do this at line 60 in this file as well, in the GetNodeID() call

travisCI currently failing with

The command "eval go get -v -t ./... " failed. Retrying, 2 of 3.
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
gecko_client/admin_api.go:60:59: not enough arguments in call to api.rpcRequester.makeRpcRequest
	have (string, string)
	want (string, string, map[string]interface {})
The command "eval go get -v -t ./... " failed. Retrying, 3 of 3.
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
github.com/kurtosis-tech/ava-e2e-tests/gecko_client
gecko_client/admin_api.go:60:59: not enough arguments in call to api.rpcRequester.makeRpcRequest
	have (string, string)
	want (string, string, map[string]interface {})

Reply:
fixt